### PR TITLE
Ajoute une route pour authentifier les contributeurs de Netlify

### PR DESCRIPTION
**Pull request by QuentinMadura**

### CODE REVIEW

File: backend/controllers/contributions.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const axios = require("axios")
+const config = require("../config/index")
+
+const AuthContribution = {
+  async verifyIdentity(token) {
+    return await axios.get(
```
Comment (desoindx) [inline backend/controllers/contributions.js:6]: no need for await here as we are returning the promise

File: backend/controllers/contributions.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      `${config.netlifyContributionURL}/.netlify/identity/user`,
```
Comment (desoindx) [inline backend/controllers/contributions.js:7]: weird to have a . in the url

File: backend/routes/contributions.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "https://test-contribution.netlify.app",
+  "https://contribuer-aides-jeunes.netlify.app",
+]
+
+module.exports = function (api) {
+  // parse application/x-www-form-urlencoded
+  api.use(urlencoded({ extended: false }))
+  api
+    .route("/contribution/auth")
+    .post(cors({ origin: ALLOWED_URLS }), AuthContribution.verify)
```
Comment (desoindx) [inline backend/routes/contributions.js:15]: shouldn't we just allow config.netlifyContributionURL ?

**Comment by QuentinMadura** [comment]
Pour tester cette PR, attention à bien ajouter la variable d'environnement NETLIFY_CONTRIBUTION_URL="https://test-contribution.netlify.app"

File: backend/controllers/contributions.js (hunk 2 above)
Comment (QuentinMadura) [reply backend/controllers/contributions.js:7]: It's an API route given by Netlif,  I cannot change anything in the URL.

File: backend/routes/contributions.js (hunk 3 above)
Comment (QuentinMadura) [reply backend/routes/contributions.js]: Thanks 👍

File: backend/controllers/contributions.js (hunk 1 above)
Comment (QuentinMadura) [reply backend/controllers/contributions.js]: Thanks 👍

**Comment by QuentinMadura** [comment]
Merci @desoindx pour la review

**Review by Kout95 — APPROVED** [review]

**Pull request merged**